src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -107,7 +164,10 @@ public UnorderedCallAssertion MustHaveHappened(Repeated repeatConstraint)
         }
 
         public class ReturnValueConfiguration<TMember>
-            : IAnyCallConfigurationWithReturnTypeSpecified<TMember>, ICallCollectionAndCallMatcherAccessor
+            : IAnyCallConfigurationWithReturnTypeSpecified<TMember>,
+              IAfterCallSpecifiedWithOutAndRefParametersConfiguration<IReturnValueConfiguration<TMember>>,

```

Comment:
(I know, the interface is weird... it took me a while to wrap my head around it)